index.js

```diff
@@ -1,7 +1,56 @@
 // Iteration 1: Names and Input
+console.log("I`m ready");
 
+let hacker1 = "The driver's name is Leal";
+let hacker2 = "The navigator's name is Duarte";
 
 // Iteration 2: Conditionals
 
+let driverName = hacker1.slice(21);
+let navigatorName = hacker2.slice(24);
+
+
+
+let numOfCharDriver = driverName.length;
+let numOfCharNavigator = navigatorName.length;
+
+
+if( driverName.length > navigatorName.length){
```

Comment:
Great job on creating the variables `numOfCharDriver` and `numOfCharNavigator `. Here on the if statement, you could've used them like:

`if( numOfCharDriver > numOfCharNavigator)`